### ISSUE TITLE
Publish TypeScript types

### ISSUE THREAD

**Issue by jelhan** [josemarluedke/ember-focus-trap#86]
It would be great if this addon could publish TypeScript types.

**Comment by josemarluedke** [comment]
I agree. Do you have time to work on it? Possibly migrating the addon to Typescript?

**Comment by jelhan** [comment]
I cannot commit to it at this point. I don't have much experience converting v2 addon to TypeScript. And would prioritize on Ember Bootstrap TypeScript conversion at this point.

**Comment by elidupuis** [comment]
For a temporary fix see https://github.com/josemarluedke/ember-focus-trap/pull/95#issuecomment-3916668740.

**Comment by josemarluedke** [comment]
This is now published in ember-focus-trap@1.2.0

**Comment by SanderKnauff** [comment]
It seems that the types are exported incorrectly. 
The focus-trap modifier is exposed as a function that takes no parameters, and does not return a modifier.

`focus-trap.d.ts` contains the following:

```ts
declare function focusTrapModifier(): {};
export default focusTrapModifier;
//# sourceMappingURL=focus-trap.d.ts.map
```
